tests/colorConvertHslToHex.phpt

```diff
@@ -54,19 +54,12 @@ $colorsToConvert = array(
         'hex' => 'ffffff',
         'hsl' => $white,
     ],
-);
-
+];
 
 foreach ($colorsToConvert as $color) {
-    $hsl = $color['hsl'];
-    $hex = $color['hex'];
-
-    $answer = Color::hslToHex($hsl);
     Assert::same(
-        $hex,
-        $answer,
-        'Incorrect hex result: "' . json_encode($hsl) .
-        '" should convert to "' . $hex .
-        '", but output was: "' . $answer . '".'
+        $color['hex'],
+        Color::hslToHex($color['hsl']),
+        'Incorrect hex result'
```

Comment:
- Please revert this foreach loop on this `phpt` file.
- It's failed on GitHub action running because running the tester should specify the `php.ini` file with loading JSON extension and the tester will run successfully.